docs/guides/quick-start.md

```diff
@@ -0,0 +1,98 @@
+## Quick Start
```

Comment:
We should avoid duplication of docs. I'm proposing then to remove Quick start from the root readme (as I did in #1084).